packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -187,6 +209,26 @@ const api = new HttpApi(stack, 'HttpProxyProdApi', {
 });
 ```
 
+To set up Mutual TLS for a custom domain
+
+```ts
+const dn = new DomainName(stack, 'DomainName', {
+      domainName: 'example.com',
+      domainNameConfigurations: [
+        {
+          certificate: Certificate.fromCertificateArn(stack, 'cert', 'arn:aws:acm:us-east-1:111111111111:certificate'),
+          endpointType: EndpointType.REGIONAL,
+          securityPolicy: 'TLS_1_2',
+        },
+      ],
+      mtls: {
```

Comment:
Why do we call it domainNameConfigurations but not mtlsConfiguration?

Reply:
Updated to mutualTlsConfiguration